config/bridge/config.toml

```diff
@@ -48,7 +48,9 @@ BridgeVersion = "v1"
     MaxConns = 20
 
 [NetworkConfig]
-GenBlockNumber = 75
-PolygonBridgeAddress = "0x4C739372258826995C302CD655beE12689B97d3F"
-PolygonZkEVMGlobalExitRootAddress = "0xB7f8BC63BbcaD18155201308C8f3540b07f84F5e"
-L2PolygonBridgeAddresses = ["0x4C739372258826995C302CD655beE12689B97d3F"]
\ No newline at end of file
+GenBlockNumber = 1
```

Comment:
can we figure out what's the right number to put here? should it be the same as `GenesisBlock` below?

Reply:
yes, it should be the same as `GenesisBlock` in the `dac/config.toml`.
https://github.com/0xPolygonHermez/zkevm-bridge-service/blob/v0.4.2/synchronizer/synchronizer.go#L115